README.md

```diff
@@ -74,4 +91,25 @@ You should see output similar to the following:
  * Debugger is active!
  * Debugger PIN: 226-556-590
 ```
+
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running the Tests
+
+Use the following command to run the unit and integration tests associated with the app:
+
+```bash
+$ poetry run pytest
+```
+
+Alternatively, if you have chosen to install Node, you may run the following instead:
+
+```bash
+$ npm run test:pytest
+```
+
+This project uses Cypress for some basic end-to-end testing. To run these tests, you will need to first start the app, and then use the following command:
```

Comment:
Thanks for the clear instructions, I've never actually used Cypress but this looks really cool!

One potential disadvantage over the suggested selenium approach is that this relies on the app already running and so misses out on the suggested automated test set up which includes creating a fresh throw-away Trello Board; so a bad test could easily end up leaving artefacts on the existing (hopefully at least dev/test!) board. With a more normal form of persistence we might solve that by spinning up the app against a throw-away database in a container or similar, but with Trello that might be more awkward. Not one to change this for, but worth bearing in mind!

Reply:
This was a drawback I was a bit unhappy with too. I decided not to prioritise sorting separate environments in the end, but it's something I would have liked to have done and may yet investigate more